solidus_importer.gemspec

```diff
@@ -17,7 +17,7 @@ Gem::Specification.new do |spec|
   spec.metadata['source_code_uri'] = 'https://github.com/solidusio-contrib/solidus_importer'
   spec.metadata['changelog_uri'] = 'https://github.com/solidusio-contrib/solidus_importer/releases'
 
-  spec.required_ruby_version = Gem::Requirement.new('~> 2.5')
+  spec.required_ruby_version = Gem::Requirement.new('> 2.5')
```

Comment:
```suggestion
  spec.required_ruby_version = Gem::Requirement.new('>= 2.5')
```

Reply:
Oh no, sorry. I did not mean to do that. Good catch.